manifests/operator-manifests/mattermost/crds/mm_clusterinstallation_crd.yaml

```diff
@@ -34,83 +34,46 @@ spec:
     name: v1alpha1
```

Comment:
was there something wrong with having the lines wrapped in this file?

Reply:
All these files are auto-generated so it seems like the generator behavior changed since the last version.